database-plugins/widgets/Database-batchsource.json

```diff
@@ -159,6 +159,16 @@
             ],
             "default": "TRANSACTION_SERIALIZABLE"
           }
+        },
+        {
```

Comment:
Can we put this in a separate Group? So that users don't confuse this as something to replace the pattern in the data itself. Something like Column Names Normalization?